terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -167,6 +179,22 @@ resource "kubernetes_cron_job" "workflow_manager" {
               args = [
                 "--k8s-namespace", var.kubernetes_namespace,
                 "--input-bucket", var.ingestion_bucket,
+                "--bsk-secret-name", kubernetes_secret.batch_signing_key.metadata[0].name,
+                "--pdks-secret-name", var.packet_decryption_key_kubernetes_secret,
```

Comment:
I think this will be fine for Narnia, but at some point, we have to work out how to provide _all_ the packet decryption keys to facilitator so it can try them all.

Reply:
I figure we can just cram them all into the one secret, comma-separated like the flag wants.